schedule2.html

```diff
@@ -0,0 +1,266 @@
+---
```

Comment:
This version is more responsive, but it doesn't handle the lightning talks/main talk crossover very well (need something akin to rowspan here)